src/components/UserList/ActionButtons.tsx

```diff
@@ -25,14 +25,20 @@ export const ActionButtons: React.FunctionComponent = () => {
             onClick={() => {
               usersService
                 .remove(ids)
-                .then(() => {
-                  enqueueSnackbar(`Removed`, {
-                    variant: "success",
-                  });
-                  userDispatch({
-                    type: "getAll",
-                    payload: userList.filter((user) => !ids.includes(user.id)),
-                  });
+                .then((value) => {
+                  if (value.toString().trim().length === 0) {
+                    enqueueSnackbar(`Removed`, {
+                      variant: "success",
+                    });
+                    userDispatch({
+                      type: "getAll",
+                      payload: userList.filter((user) => !ids.includes(user.id)),
+                    });
+                  } else {
+                    enqueueSnackbar(`You cannot delete yourself.`, {
```

Comment:
consider showing error in case `ids` contains current user `id` and not sending request with error or filtering it out from array showing warning that current user id was not taken into request